src/app/Shared/FeatureFlag/FeatureFlag.tsx

```diff
@@ -71,11 +71,13 @@ export interface DynamicFeatureFlagProps {
 export const DynamicFeatureFlag: React.FunctionComponent<DynamicFeatureFlagProps> = ({ levels, component }) => {
   return (
     <>
-      {levels.map((level) => (
-        <FeatureFlag strict level={level}>
-          {component(level)}
-        </FeatureFlag>
-      ))}
+      {levels
+        .filter((level) => levels.includes(level))
```

Comment:
This is not needed right? As it always returns true?

Reply:
Oh yea, whoops - I was thinking of doing something else with a refactoring and then left this half-baked thought in, lol.